application/src/test/resources/speedtest/travelSearch-expected-results-bt.csv

```diff
@@ -1,3 +1,3 @@
 tcId,nTransfers,duration,cost,walkDistance,startTime,endTime,agencies,modes,routes,stops,details
-1,0,1h25s,0,0,12:00:00,13:00:25,,,,,Unknown transit 0tx 1h25s
-2,0,1h25s,0,0,12:00:00,13:00:25,,,,,Unknown transit 0tx 1h25s
+1,0,1h22s,0,0,12:00:00,13:00:22,,,,,Unknown transit 0tx 1h22s
+2,0,1h22s,0,0,12:00:00,13:00:22,,,,,Unknown transit 0tx 1h22s
```

Comment:
Do you know why all of these speed tests are 3 seconds quicker? Ideally changing the internal angle representation should not change travel time.

Reply:
It is because of the inAngle / outAngle fix for creating the temporary edge has changed the turn time near the origin / destination.